### PR TITLE
Fix HTML errors

### PR DESCRIPTION
**Pull request by MichaelDeciso**
This fixes some typos, deprecated elements and the usage of the `alt` attribute on `<div>` elements (which is not allowed and does not work either).

### CODE REVIEW

File: src/www/diag_ipsec_spd.php [hunk 1]
```diff
@@ -73,9 +73,9 @@
                 <tr>
                   <td colspan="2"></td>
                   <td colspan="3">
-                    <span class="fa fa-arrow-right fa-fw" aria-hidden="true" alt="in"></span>
+                    <span class="fa fa-arrow-right fa-fw" aria-hidden="true"></span>
                     <?= gettext("incoming (as seen by firewall)"); ?> <br/>
-                    <span class="fa fa-arrow-left fa-fw" aria-hidden="true" alt="out"></span>
+                    <span class="fa fa-arrow-left fa-fw" aria-hidden="true"></span>
```
Comment (fabianfrz) [inline src/www/diag_ipsec_spd.php]: Probably title was meant -> creates a tooltip

File: src/www/diag_limiter_info.php [hunk 2]
```diff
@@ -68,7 +68,7 @@ function getlimiteractivity() {
       });
   }
   function activitycallback(transport) {
-    jQuery('#limiteractivitydiv').html('<font face="Courier" size="2"><pre style="text-align:left;">' + transport.responseText  + '<\/pre><\/font>');
+    jQuery('#limiteractivitydiv').html('<pre style="text-align:left;">' + transport.responseText  + '<\/pre>');
```
Comment (fabianfrz) [inline src/www/diag_limiter_info.php]: The backslash is probably not needed and the pre would make more sense when it is not generated in js if possible

File: src/www/services_dhcp.php [hunk 3]
```diff
@@ -1071,7 +1071,7 @@ function removeRow() {
                             <tfoot>
                               <tr>
                                 <td colspan="4">
-                                  <div id="addNew" style="cursor:pointer;" class="btn btn-default btn-xs" alt="add"><i class="fa fa-plus fa-fw"></i></div>
+                                  <div id="addNew" style="cursor:pointer;" class="btn btn-default btn-xs"><i class="fa fa-plus fa-fw"></i></div>
```
Comment (fabianfrz) [inline src/www/services_dhcp.php]: Very likely title was meant - also gettext is missing

File: src/www/wizard.php [hunk 4]
```diff
@@ -885,7 +885,7 @@ function showchange() {
 				echo $field['typehint'];
 			}
 			if($field['warning'] <> "") {
-				echo "<br /><b><font color=\"red\">" . $field['warning'] . "</font></b>";
+				echo "<br /><b><span style='color: red;'>" . $field['warning'] . "</span></b>";
```
Comment (fabianfrz) [inline src/www/wizard.php]: Use a suitable bootstrap class instead

File: src/www/diag_ipsec_spd.php (hunk 1 above)
Comment (MichaelDeciso) [reply src/www/diag_ipsec_spd.php]: I know `title` creates a tooltip, but I didn't think one was intended - it's all in lower case.
Of course, I could add one now - but what shall I take as text? And shall I add gettext as well?

File: src/www/diag_limiter_info.php (hunk 2 above)
Comment (MichaelDeciso) [reply src/www/diag_limiter_info.php]: I think you're right.

File: src/www/wizard.php (hunk 4 above)
Comment (MichaelDeciso) [reply src/www/wizard.php]: The most suitable would be `text-warning`, though that would be orange, not red. Is that OK?

Comment (fichtner) [reply src/www/wizard.php]: for this case text-alert would be more appropriate despite the "warning" designation

Comment (MichaelDeciso) [reply src/www/wizard.php]: text-alert is not defined in any CSS file. Did you mean "text-info"?

Comment (fichtner) [reply src/www/wizard.php]: sorry, I meant text-danger

Comment (MichaelDeciso) [reply src/www/wizard.php]: Hm, would that really be more appropriate? I thought this field showed warnings that weren't that urgent? To me, it seems that classifying them as 'danger' would be a bit overkill.

Comment (fichtner) [reply src/www/wizard.php]: ok, let's back up a little :) how often is 'warning' used in the wizard xml files?

Comment (MichaelDeciso) [reply src/www/wizard.php]: I cannot find any occurrence in `wizard/system.xml` and `wizard/openvpn.xml`.

If you really think text-danger is more appropriate, I'll just go with that. :)

Comment (fichtner) [reply src/www/wizard.php]: if nothing uses it the point is moot. better remove the "functionality" ?

Comment (MichaelDeciso) [reply src/www/wizard.php]: You know the system better than I do. Are the two XML files I mentioned the only places this "warning" could come from?

I'll leave it to you to decide whether to remove it. :)

Comment (fichtner) [reply src/www/wizard.php]: yeah, only those two are left with the wizard system deprecated and in need of a better replacement eventually. please remove then, thanks!

File: src/etc/inc/interfaces.inc [hunk 5]
```diff
@@ -4357,7 +4357,7 @@ function get_interfaces_info()
         if ($bridge) {
             $bridge_text = `/sbin/ifconfig {$bridge}`;
             if (stristr($bridge_text, "blocking") <> false) {
-                $ifinfo['bridge'] = "<b><font color='red'>" . gettext("blocking") . "</font></b> - " . gettext("check for ethernet loops");
+                $ifinfo['bridge'] = "<b><span style='color: red'>" . gettext("blocking") . "</span></b> - " . gettext("check for ethernet loops");
```
Comment (fichtner) [inline src/etc/inc/interfaces.inc]: how about this one in text-danger then? :D

Comment (MichaelDeciso) [reply src/etc/inc/interfaces.inc]: As you wish ;)

File: src/www/widgets/widgets/carp_status.widget.php [hunk 6]
```diff
@@ -54,13 +54,13 @@
 <?php
       if (get_single_sysctl('net.inet.carp.allow') <= 0 ) {
           $status = gettext("DISABLED");
-          echo "<span class=\"fa fa-remove fa-fw text-danger\" title=\"$status\" alt=\"$status\" ></span>";
+          echo "<span class=\"fa fa-remove fa-fw text-danger\" title=\"$status\" ></span>";
```
Comment (fichtner) [inline src/www/widgets/widgets/carp_status.widget.php]: does this still display tooltips?

File: src/www/vpn_openvpn_server.php [hunk 7]
```diff
@@ -1077,7 +1077,7 @@
                     </tr>
                     <tr>
                       <td style="width:22%" id="ipv4_tunnel_network"><a id="help_for_tunnel_network" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("IPv4 Tunnel Network"); ?></td>
-                      <td witdh="78%">
+                      <td width="78%">
```
Comment (fichtner) [inline src/www/vpn_openvpn_server.php]: fixed the remaining ones via https://github.com/opnsense/core/commit/1bfc8ef5df

File: src/www/widgets/widgets/carp_status.widget.php (hunk 6 above)
Comment (MichaelDeciso) [reply src/www/widgets/widgets/carp_status.widget.php]: `title` is the attribute that displays tooltips. `alt` is only used when an img(!) cannot load (except in IE, which interprets the `alt` as `title` if no `title` is specified).

File: src/www/vpn_openvpn_server.php (hunk 7 above)
Comment (fichtner) [reply src/www/vpn_openvpn_server.php]: maybe this and the other one should be changed to style= as well?

$ git grep td.witdh
src/www/vpn_openvpn_server.php:                      <td witdh="78%">
$ git grep td.width
src/www/interfaces.php:                          <td width="22%"><i class="fa fa-info-circle text-muted"></i> <?=gettext('Service Provider') ?></td>

Comment (MichaelDeciso) [reply src/www/vpn_openvpn_server.php]: Done.

**Comment by fichtner** [comment]
merged, many thanks!

**Pull request merged**